exec/java-exec/src/main/java/org/apache/drill/exec/store/pojo/PojoDataType.java

```diff
@@ -36,6 +36,7 @@
 
   private final List<SqlTypeName> types = Lists.newArrayList();
   private final List<String> names = Lists.newArrayList();
+  private final List<Boolean> nullables = Lists.newArrayList();
```

Comment:
Again, single list of objects rather than three correlated list of primitives?

Reply:
RecordDataType uses the `names` and `types` lists separately. So, I'll need atleast 2 lists.